yarn.lock

```diff
@@ -0,0 +1,1363 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Why is the yarn.lock file present in reactive-manual? If you can remove this, this PR is good to go.